site/app/libraries/Access.php

```diff
@@ -260,7 +260,7 @@ private function loadDirectories() {
         ];
         $this->directories["submissions"] = [
             "base" => $this->core->getConfig()->getCoursePath() . "/submissions",
-            "subparts" => ["gradeable", "submitter", "version"],
+            "subparts" => ["gradeable", "submitter"],
```

Comment:
By only doing this, a student could now access this file which is not intended behavior. Instead a new element needs to be added to directories for the higher level directory.